City-of-Helsinki/drupal-hdbt
# UHF-9088: Translations

# [UHF-9088](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9088)


## What was done


* Updated translations based on changes via Strategia instance.

## How to install and test

Check instructions from [helfi platform config](https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/706).

## Other PRs


* https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/170
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/918
* https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/pull/152
* https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/275

[UHF-9088]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9088?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
